MainWindow.py

```diff
@@ -1,5 +1,8 @@
+# Mikcy: The MainWindow file name is a bit of a mis nomenclature. it a left over from the old code where this module actually lanuceched
+#        the main gui window. I know this sounds like a pure nuisance but remember a code is written only once and used (read, maintained, changed)
+#        by generations of programmers. Names are important for understanding. Care to try another name?
 from ExperimentManager1 import ExperimentManager
-from experimentgui import ExperimentGUI, OpponentType
+from experimentgui import ExperimentGUI, OpponentType     # Micky: I do not understand how this works. OpponenType is not defined in experimentGui
```

Comment:
yes but it inherits it from an import it has

Reply:
OK. Got it :-)